Jenkinsfile

```diff
@@ -124,8 +124,8 @@ node {
         stage("clang-debug:addr-ub-sanitizers") {
           if (env.BRANCH_NAME == 'master' || full_ci) {
             sh "export CCACHE_BASEDIR=`pwd`; cd clang-debug-addr-ub-sanitizers && make hyriseTest hyriseSystemTest -j \$(( \$(cat /proc/cpuinfo | grep processor | wc -l) / 3))"
-            sh "LSAN_OPTIONS=suppressions=.lsan-ignore.txt ASAN_OPTIONS=suppressions=.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseTest clang-debug-addr-ub-sanitizers"
-            sh "LSAN_OPTIONS=suppressions=.lsan-ignore.txt ASAN_OPTIONS=suppressions=.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseSystemTest clang-debug-addr-ub-sanitizers"
+            sh "LSAN_OPTIONS=suppressions=artifacts/ci/.lsan-ignore.txt ASAN_OPTIONS=suppressions=artifacts/ci/.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseTest clang-debug-addr-ub-sanitizers"
+            sh "LSAN_OPTIONS=suppressions=artifacts/ci/.lsan-ignore.txt ASAN_OPTIONS=suppressions=artifacts/ci/.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseSystemTest clang-debug-addr-ub-sanitizers"
```

Comment:
For me, "artifacts" are things created as part of the build process. Maybe "resources"?

Reply:
The only other thing I can think of right now is "misc", but that doesn't mean anything, so "resources" it is, for now.